plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -388,6 +375,25 @@ class NewCreateServiceModal extends Component {
     const {serviceFormErrors, serviceSpec} = this.state;
     let validationErrors = [];
 
+    const APP_VALIDATORS = Hooks.applyFilter(
```

Comment:
We we really want to allow the removal of validators? Let's use and _action_ hook instead.